debian/rules

```diff
@@ -213,6 +213,19 @@ override_dh_auto_install:
 	cp -ar factory/* debian/tmp
 	mkdir -p debian/tmp/usr/bin
 	mkdir -p debian/tmp/usr/lib64
+
+	# splash functionality
+	mkdir -p debian/tmp/usr/share/plymouth/themes/
+	cp -a vendor/plymouth-theme-ubuntu-core/ubuntu-core \
+		debian/tmp/usr/share/plymouth/themes/
+	mkdir -p debian/tmp/usr/share/fonts/
+	cp /usr/share/fonts/truetype/ubuntu/Ubuntu-R.ttf \
+		debian/tmp/usr/share/fonts/Plymouth.ttf
+	# We need the seat rules for drm. Replace jinja template bits for that.
+	sed -e '/{% raw [ -]%}/d' -e '/{% endraw %}/d' -e 's#{{ROOTBINDIR}}#/bin#' \
```

Comment:
I do think we should do that. Could we instead enable logind and filter the installed binaries?

Reply:
ok, this is done now